pkg/meta/redis.go

```diff
@@ -2893,7 +2893,65 @@ func (m *redisMeta) scanDeletedSlices(ctx Context, scan deletedSliceScan) error
 	return nil
 }
 
-func (m *redisMeta) scanDeletedFiles(ctx Context, scan deletedFileScan) error {
+func (m *redisMeta) scanPendingSlices(ctx Context, scan pendingSliceScan) error {
+	if scan == nil {
+		return nil
+	}
+
+	pendingKeys := make(chan string, 1000)
+	c, cancel := context.WithCancel(ctx)
+	defer cancel()
+	go func() {
+		_ = m.hscan(c, m.sliceRefs(), func(keys []string) error {
+			for i := 0; i < len(keys); i += 2 {
+				pendingKeys <- keys[i]
```

Comment:
keys[i+1] is the value, we should check the value here

Reply:
Done